app/components/works/globus_setup_component.html.erb

```diff
@@ -15,10 +15,16 @@
       </div>
    </div>
 <% else %>
-   <div class="row my-5 mx-2 alert alert-info">
+   <div class="row my-5 mx-2 alert alert-warning">
       <div class="col-12">
-         <p>Upload your files to the Globus folder "<%=work_version.globus_endpoint%>" and then click the "Edit or Deposit" button on this page to complete your deposit.
-         Refer to our <%= link_to 'help page', Settings.globus.help_doc_url %> if you have questions, or contact us at the Help link above.</p>
+         <p><strong><span class="fa fa-exclamation-circle me-2"></span> Almost there! Just a few more steps. If you need help, visit the
+           <%= link_to 'help page', "#{Settings.globus.help_doc_url}/edit?pli=1#heading=h.f2rvtrjiw01}", target: '_blank' %>.</strong></p>
```

Comment:
My eyes might be crossed, but is there an extraneous `}` after the heading ID?

Reply:
Ack, good catch. Fixing.